src/tss2-fapi/ifapi_get_intl_cert.h

```diff
@@ -12,4 +12,11 @@ ifapi_get_intl_ek_certificate(
     unsigned char ** buffer,
     size_t *cert_size);
 
+char *
+base64_encode(const unsigned char* buffer);
```

Comment:
This should be somewhat namespaced imho. its a way to generic name otherwise.

However, if the JSON/B64 decoding is moved into ifapi_get_intl_cert these two functions can be made static.

Reply:
Both [en|de]code function changed to static